src/index.tsx

```diff
@@ -329,12 +330,12 @@ export default class TimeField extends React.Component<Props, State> {
       <input
         ref={this.inputRef}
         type="text"
-        {...props}
         value={value}
         onChange={onChangeHandler}
         onKeyDown={onKeyDownHandler}
         className={className}
         style={style}
+        autoFocus={props.autoFocus}
```

Comment:
did you get this to work integrated with subtitle-ui? I wasn't able to, maybe because:
https://github.com/dotsub/vtms-subtitle-edit-ui/blob/master/src/subtitleEdit/cues/edit/CueTextEditor.tsx#L276

Reply:
There is separate PR for subtitle-edit. https://github.com/dotsub/vtms-subtitle-edit-ui/pull/344